initfiles/componentfiles/configschema/xsd/dafilesrv.xsd

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xs:schema
-    xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormhpcc:presetValue="qualified" attributeFormhpcc:presetValue="unqualified"
```

Comment:
make sure the component owner reviews the xsd, I am not reviewing the xsd

Reply:
that is the next step. I will open a JIRA for that specifically.